tekton-pipeline/pipeline-jenkins-bump-version.yaml

```diff
@@ -0,0 +1,56 @@
+﻿apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  creationTimestamp: '2020-10-20T04:39:54Z'
+  generation: 5
+  managedFields:
+    - apiVersion: tekton.dev/v1beta1
+      fieldsType: FieldsV1
+      fieldsV1:
+        'f:spec':
+          .: {}
+          'f:params': {}
+          'f:resources': {}
+          'f:tasks': {}
+      manager: Mozilla
+      operation: Update
+      time: '2020-10-20T07:48:22Z'
+  name: jenkins-bump-version
+  namespace: art-jenkins
+  resourceVersion: '66332946'
+  selfLink: >-
+    /apis/tekton.dev/v1beta1/namespaces/art-jenkins/pipelines/jenkins-bump-version
+  uid: 27c6e3e7-dca9-49a7-9c5a-ae99cf3666f4
```

Comment:
Templates should be free of stuff that's specific to the deployment or otherwise filled in after creation. I'm fairly sure you could (and should) remove everything here except `name:` (possibly `managedFields` but that looks empty anyway).

Similar comment for every file below.

Reply:
yes, I just downloaded the existing script